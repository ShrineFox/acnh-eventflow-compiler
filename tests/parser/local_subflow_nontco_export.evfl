flow Root():
    if TestActor1.Query0() == 0:
        TestActor1.Action0()
        run Local()
    else:
        TestActor1.Action0()
        run Local()
    run Local()

local flow Local():
    TestActor1.Action0()
