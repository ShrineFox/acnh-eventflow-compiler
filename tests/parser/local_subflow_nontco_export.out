RootNode[name=Local, vardefs=[], out_edges=[Event13]]
RootNode[name=Root, vardefs=[], out_edges=[Event9]]
ActionNode[name=Event0, action=EventFlowActionAction0(), params={}, out_edges=[Event1]]
SubflowNode[name=Event1, ns=, called_root_name=Local, params={}, out_edges=[Event13]]
ActionNode[name=Event4, action=EventFlowActionAction0(), params={}, out_edges=[Event5]]
SubflowNode[name=Event5, ns=, called_root_name=Local, params={}, out_edges=[Event13]]
SwitchNode[name=Event9, query=EventFlowQueryQuery0(), params={}, cases={'Event0': [0], 'Event4': [1, 2]}, out_edges=[Event0, Event4]]
ActionNode[name=Event13, action=EventFlowActionAction0(), params={}, out_edges=[]]
